index.js

```diff
@@ -0,0 +1,147 @@
+import { digits, keys, controls } from './src/keyboard.js';
+
+const wrapper = document.createElement('div');
+wrapper.className = 'wrapper';
+
+document.body.append(wrapper);
+
+const input = document.createElement('textarea');
+
+wrapper.append(input);
+
+const keyboard = document.createElement('div');
+keyboard.className = 'keyboard-container';
+wrapper.append(keyboard);
+
+let language = localStorage.getItem('lang') || 'eng';
+let isCapsActive = false;
+let isShift = false;
+
+
+function createKeys(keysArr, keyClassName, keyValue) {
+  for (let i = 0; i < keysArr.length; i += 1) {
+    const element = document.createElement('div');
+    element.setAttribute('class', 'key');
+    element.classList.add(keyClassName);
+    element.setAttribute('id', keysArr[i].code);
+    element.innerText = keyClassName === 'letter' ? keysArr[i][keyValue][0] : keysArr[i][keyValue];
+    keyboard.appendChild(element);
+  }
+}
+
+createKeys(digits, 'digit', 'digit');
+createKeys(keys, 'letter', language);
+createKeys(controls, 'control', 'value');
+
+document.getElementById('lang').innerText = language.toUpperCase();
+
+function toggleLang() {
+  if (language === 'eng') {
+    document.querySelectorAll('.key.letter').forEach((e, i) => {
+      e.innerText = isCapsActive ? keys[i].ru[0].toUpperCase() : keys[i].ru[0];
+    });
+    language = 'ru';
+    localStorage.setItem('lang', 'ru');
+    document.getElementById('lang').innerText = 'RU';
+  } else {
+    document.querySelectorAll('.key.letter').forEach((e, i) => {
+      e.innerText = isCapsActive ? keys[i].eng[0].toUpperCase() : keys[i].eng[0];
+    });
+    language = 'eng';
+    localStorage.setItem('lang', 'eng');
+    document.getElementById('lang').innerText = 'ENG';
+  }

```

Comment:
40-45 and 47-52 almost the same rows(It is good case in order to create a function)